src/main/kotlin/no/nav/personbruker/dittnav/eventtestproducer/beskjed/beskjedApi.kt

```diff
@@ -14,4 +15,16 @@ fun Route.beskjedApi(beskjedProducer: BeskjedProducer) {
         }
     }
 
+    post("/produce/multiple/beskjed/{number}") {
```

Comment:
Noen grunn for at denne bare finnes for Beskjed og ikke de andre typene?

Reply:
Dette er noe som har sneket seg med som jeg har brukt til lokal testing 🙈